CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required( VERSION 3.2.2 )
+project( libsnd )
+
+### Standard
+set( CMAKE_CXX_STANDARD 14 )
+set( CMAKE_CXX_STANDARD_REQUIRED ON )
+set( CMAKE_CXX_EXTENSIONS OFF )
+
+### Verbosity
+set( CMAKE_COLOR_MAKEFILE ON )
+set( CMAKE_VERBOSE_MAKEFILE ON )
+
+### Optimizations
+if( MSVC )
+	add_compile_options( /W4 )
+elseif( CMAKE_COMPILER_IS_GNUCXX )
+	add_compile_options( -O2 )
```

Comment:
Just wondering: cmake should already apply O2 to release builds. Any reason to manually set that flag for all configurations?

Reply:
No particular reason I suppose after hearing that it is on for release builds. I haven't really touched the `-DCMAKE_BUILD_TYPE=Release`-type flags of cmake myself.